code/game/objects/items/devices/radio/radio_objects.dm

```diff
@@ -139,6 +139,14 @@ GLOBAL_LIST_EMPTY(deadsay_radio_systems)
 	to_chat(user, "<span class='notice'>You <b>[broadcasting ? "enable" : "disable"]</b> [src]'s hotmic!</span>")
 	add_fingerprint(user)
 
+/obj/item/radio/CtrlShiftClick(mob/user)
+	if(user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user) || !istype(user))
```

Comment:
The hands blocked trait is added when mobs becomes unconcious or dead, so this check is not needed.
```suggestion
	if(HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user) || !istype(user))
```